src/css/welcome.css

```diff
@@ -523,7 +523,69 @@ button {
   }
 }
 
+.extension-promo {
+  background: pink;
+
+  position: fixed;
+  bottom: 16px;
+  right: 16px;
+
+  background: linear-gradient(
+      139.76deg,
+      rgba(255, 255, 255, 0.5) 71.26%,
+      rgba(76, 84, 210, 0.0001) 115.29%
+    ),
+    #ffffff;
```

Comment:
dark mode styling required

Reply:
done in https://github.com/brave/brave-talk/pull/174/commits/c6be8e121fa9f4f35ad23a3714d26029f1ca6b96